### DISCUSSION TITLE
quick/request-input

### DISCUSSION THREAD

**Discussion by giscus** [gogf/gf-site#121] [General]
# quick/request-input

使用GoFrame框架在Web Server中获取客户端提交的请求参数，重点讲解通过r.Get方法处理Query String、Form和Body等HTTP方法提交的参数。学习内容包括参数默认值处理及参数类型自动识别。通过示例代码详细展示如何在GoFrame中接收和处理参数，并对常见问题进行分析，为后续章节对参数对象的结构化处理奠定基础。

https://goframe.org/quick/request-input

**Comment by Nexzk** [comment]
good
